internal/service/alarms/api/overlay.yaml

```diff
@@ -0,0 +1,53 @@
+overlay: 1.0.0
+info:
+  title: Internal Endpoints Overlay
+  version: 1.0.0
+actions:
+  - target: "$.paths"
+    update:
+      /internal/v1/caas-alerts/alertmanager:
+        post:
+          operationId: AmNotification
+          summary: Receive Alertmanager notifications
```

Comment:
curiousity nit: why isn't this camelcase, ie. AlertManager, throughout, such as the schema defs (line 20 below)? Not asking for it to be changed, if it's intentional, or that it should be changed as part of this PR since it's pre-existing and unrelated to this PR

Reply:
Yeah, it's absolutely unrelated to this PR. It just struck me as odd.